src/components/OSCALMarkdownProse.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { OSCALMarkdownProse } from "./OSCALMarkdownProse";
```

Comment:
```suggestion
import OSCALMarkdownProse from "./OSCALMarkdownProse";
```
Right now the component tests are failing and I believe if you remove the brackets around `OSCALMarkdownProse` it will prevent them from failing